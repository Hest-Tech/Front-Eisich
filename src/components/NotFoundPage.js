/**
 * This file contains Not found Page component
 */

import React from 'react';

const NotFoundPage = () => (
    <div className="help-page-container"><h1>404!</h1></div>
);

export default NotFoundPage;