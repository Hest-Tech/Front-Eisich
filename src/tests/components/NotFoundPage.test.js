import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../pages/NotFoundPage';


test('Should render NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});