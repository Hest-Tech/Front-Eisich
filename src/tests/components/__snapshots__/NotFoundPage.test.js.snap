// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render AboutPage correctly 1`] = `
<div
  className="help-page-container"
>
  About Us
</div>
`;

exports[`Should render NotFoundPage correctly 1`] = `
<div
  className="help-page-container"
>
  <h1>
    404!
  </h1>
</div>
`;
